### ISSUE TITLE
TwoFactorBackupable depends on Devise::DatabaseAuthenticatable.bcrypt

### ISSUE THREAD

**Issue by bsedat** [tinfoil/devise-two-factor#22]
From #18. We should look into separating the bcrypt functionality into a helper (as mentioned on https://github.com/tinfoil/devise-two-factor/blob/75d9ce56b8b39c8257d0d3729f8cc64bcf824f66/lib/devise_two_factor/models/two_factor_backupable.rb#L42)

**Comment by bsedat** [comment]
Got this in Devise 3.5.1: 
```
DEPRECATION WARNING: Devise.bcrypt is deprecated; use Devise::Encryptor.digest instead. (called from block in generate_otp_backup_codes! at /Users/bsedat/workspace/devise-two-factor/lib/devise_two_factor/models/two_factor_backupable.rb:28)
```

**Comment by bsedat** [comment]
Fixed in #36
